packages/explorable-explanations/src/lib/utils.js

```diff
@@ -190,44 +216,46 @@ utils.setupInterestGroupCanvas = (p) => {
   overlayCanvas.parent('interest-canvas');
   overlayCanvas.style('z-index', 2);
   p.textSize(config.canvas.fontSize);
+  // eslint-disable-next-line no-undef
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
What does this do for dev mode?

Reply:
Using a custom set env variable that will decide if canvas is running standalone or with not.